trust-quorum/src/coordinator_state.rs

```diff
@@ -137,39 +145,76 @@ impl CoordinatorState {
     // This method is "in progress" - allow unused parameters for now
     #[allow(unused)]
     pub fn send_msgs(&mut self, now: Instant, outbox: &mut Vec<Envelope>) {
-        match &self.op {
-            CoordinatorOperation::CollectShares {
-                epoch,
-                members,
-                collected_shares,
-                ..
-            } => {}
-            CoordinatorOperation::CollectLrtqShares { members, shares } => {}
-            CoordinatorOperation::Prepare { prepares, prepare_acks } => {
-                for (platform_id, prepare) in prepares.clone().into_iter() {
-                    outbox.push(Envelope {
-                        to: platform_id,
-                        from: self.reconfigure_msg.coordinator_id().clone(),
-                        msg: PeerMsg::Prepare(prepare),
-                    });
+        if now >= self.retry_deadline {
+            self.retry_deadline = now + self.reconfigure_msg.retry_timeout();
```

Comment:
For my understanding, what is the typical duration of `retry_timeout`?

For clarity, I'd consider an early return here, leaving the code below untouched.

Reply:
I think a typical retry timeout would be about 100ms.

Reply:
Good idea on the early return. Done in [50ebf64](https://github.com/oxidecomputer/omicron/pull/8000/commits/50ebf6479db5b3e8e345394a1bcf2943306edf78)